src/pages/version/Tabs.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useEffect, useMemo } from "react";
+import { Tab } from "@leafygreen-ui/tabs";
+import { useParams, useHistory, useLocation } from "react-router-dom";
+import { usePatchAnalytics } from "analytics";
+import { CodeChanges } from "components/PatchTabs/CodeChanges";
+import { DownstreamTasks } from "components/PatchTabs/DownstreamTasks";
```

Comment:
DownstreamTasks asks for `childPatches` which is currently a property of Patch but not Version. We should most likely omit rendering this tab if there is no synonym for Version

Reply:
ChildPatches gets imported from the patch field in the version query https://github.com/khelif96/spruce/blob/07d0d313e05e437ad97b448652d51f2c4c5baff7/src/pages/Version.tsx#L106 

And is passed in that same file.
 It will only be available if viewing a version that has a corresponding patch. Since it is only relevant to those types of versions it made sense to keep it as a field belonging to  a patch.